src/metax_api/settings.py

```diff
@@ -610,3 +610,8 @@
         'AUTO_APPROVER':        app_config_dict.get('REMS', {}).get('AUTO_APPROVER'),
         'FORM_ID':          int(app_config_dict.get('REMS', {}).get('FORM_ID')),
     }
+
+if executing_in_travis:
+    DRAFT_ENABLED = 'DRAFT_ENABLED'
+else:
+    DRAFT_ENABLED = app_config_dict.get('DRAFT_ENABLED', False)
```

Comment:
Use boolean values instead of strings for feature switches.

Reply:
Ok changed for Travis